app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -391,6 +388,18 @@ const CallScreen = ({
         return null;
     }
 
+    // The user should receive an alert if all of the following conditions apply:
+    // - Recording has started.
+    // - Recording has not ended.
+    // - The alert has not been dismissed already.
+    const recording = Boolean(currentCall.recState?.start_at && !currentCall.recState?.end_at);
+    if (recording && !currentCall.recAcknowledged) {
+        const topComponentId = NavigationStore.getNavigationTopComponentId();
+        if (topComponentId !== Screens.CALL) {
```

Comment:
This guards against the bar component throwing up an alert when it's behind the call screen (which would then appear after leaving the call, even if you'd accepted the call screen's alert).